index.html

```diff
@@ -913,6 +926,10 @@ <h2 class="h4">
       <p class="text-secondary">(no tracks)</p>
     <% } %>
   <% } %>
+  <div class="row mb-3">
+    <label for="edit_note-txt" googl="true" style="padding: 0;"><strong>Edit note:</strong></label>
```

Comment:
What's the `googl` attribute on the label tag supposed to do? That's not a standard tag.

I'd like this to better fit into the other bootstrap styles on the page, so use `class="form-label fw-bold"`. I think using bold via css is better than `<strong>` here, since the bold is purely for visual consistency and isn't really emphasis.